stories/Chips/Chips.stories.js

```diff
@@ -0,0 +1,222 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+
+import { Chip, ChipLabel, Button, Icon } from '../../src'
+
+import VersioneStandard from './docs/VersioneStandard.md'
+import GreatVersion from './docs/GreatVersion.md'
+import DisabledChip from './docs/DisabledChip.md'
+import ChipGroups from './docs/ChipGroups.md'
+
+const VersioneStandardComponent = () => (
+  <div className="row">
```

Comment:
Can you use the `Row` reactstrap component here?
ref: https://reactstrap.github.io/components/layout/

Reply:
Sure!